frontends/web/src/components/layout/version.tsx

```diff
@@ -14,19 +15,32 @@
  * limitations under the License.
  */
 
-import { PropsWithChildren } from 'react';
-import { load } from '../../decorators/load';
-import { translate, TranslateProps } from '../../decorators/translate';
+import { FunctionComponent, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { getVersion } from '../../api/version';
 
-interface VersionProps {
-    version: string;
-}
+const Version: FunctionComponent = () => {
+    const { t } = useTranslation();
+    const [version, setVersion] = useState<string>('');
 
-type Props = VersionProps & TranslateProps;
+    useEffect(() => {
```

Comment:
This effect could be simplified to:

```
    useEffect(() => {
        getVersion().then(setVersion);
    }, []);
```

Reply:
I could only get rid of the warning with the 2nd commit that contains the `ignore` variable and does not update the state if the component is already unmounted...

Should I include the empty array?

Reply:
Yes, that means that the dependencies of that `useEffect` are none, since `getVersion` it's a constant and `setVersion` is a setter (constant) eslint rules should not warn you anything

Reply:
As discussed I'll remove the 2nd commit that fixes the warning and change this PR without `ignore` variable and cleanup function.